plugins/trader/trade.js

```diff
@@ -0,0 +1,293 @@
+/*
+  @askmike
```

Comment:
better to link to the discussion than copy snippets of it in a comment :)

Reply:
Agreed, will update.